integration-tests/integration-test-script/world.py

```diff
@@ -448,7 +448,8 @@ def test_debts_values(self, test_traffic_pairs, time, speed, debts, all_routes,
                         node.id, intended_debts[node][owed], owed.id))
                     continue
                 if not fuzzy_match(debts[node.id][owed.id], intended_debts[node][owed]):
-                    print("{} has a predicted debt of {} for {} but actual debt is {} {:.2%} accurate".format(
+                    if debts[node.id][owed.id] != 0:
+                        print("{} has a predicted debt of {} for {} but actual debt is {} {:.2%} accurate".format(
```

Comment:
if the value is zero we should log in a different way. We don't want to just silence this message